spring-cloud-starter-stream-sink-mqtt/src/main/java/org/springframework/cloud/stream/app/mqtt/sink/MqttSinkProperties.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.stream.app.mqtt.sink;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Properties for the Mqtt Sink.
+ *
+ * @author Janne Valkealahti
+ *
+ */
+@ConfigurationProperties("mqtt")
```

Comment:
I wonder how does that work if we use the same prefix on different `@ConfigurationProperties` in the same app...
I mean that `@EnableConfigurationProperties({ MqttProperties.class, MqttSinkProperties.class })` and this `mqtt` prefix on the `MqttProperties` as well. 

Does it mean that Boot properly merge them into one set and then is able to distribute provided properties to proper classes?
Just confused a bit and would like to have some clarification.
Thanks